src/Inkluzitron/Handlers/InviteHandler.cs

```diff
@@ -45,6 +46,15 @@ private async Task OnUserJoinedAsync(SocketGuildUser user)
             var inviteLinks = await user.Guild.GetInvitesAsync();
             var inviteeDb = await dbContext.GetOrCreateUserEntityAsync(user);
 
+            try
+            {
+                await user.SendMessageAsync(Config.GetValue<string>("Welcoming:DirectMessage"));
+            }
+            catch (HttpException)
```

Comment:
```suggestion
            catch (HttpException ex) when (ex.DiscordCode == 50007)
```
Zase br br nezabíjel bych úplně všechny.